packages/members-api/lib/users.js

```diff
@@ -56,28 +56,34 @@ function listMembers(options) {
 }
 
 module.exports = function ({
-    sendEmailWithMagicLink,
     stripe,
     memberModel
 }) {
     Member = memberModel;
 
+    async function getStripeSubscriptions(member) {
```

Comment:
Both `getStripeSubscriptions` and `destroyStripeSubscriptions` are wrapping and exposing `stripe` dependent functionality. Went with exposing it here as the subscriptions are mostly used in context of the member. They should be gone once relations are properly implemented on the model layer.

Reply:
Extra point: simplifies repeated `if (stripe) ....` logic